release-notes/2022-12/README.md

```diff
@@ -0,0 +1,203 @@
+# SystemLink Enterprise 2022-12 Release Notes
+
+The 2022-12 release bundle for SystemLink Enterprise has been published to <https://niedge01.jfrog.io>. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- The License service now included in the top level helm chart. Refer to **Helm Chart Breaking Changes** for details on the required configuration for this service.
+- Users can specify data table IDs as variables in dashboards.
+- Users can customize columns, filter, save, and load views in the Products grid.
```

Comment:
```suggestion
- Users can customize, filter, save, and load views in the Products grid.
```
I know it's not as specific but it's easier to read and, I think, still gets the point across